src/components/galaciuc.less

```diff
@@ -0,0 +1,142 @@
+@import "../main";
+
+
+.galaciuc {
+  .blue-section-wrapper {
+    .blue-section-wrapper (@img_path: "../../assets/img/bag.png");
+  }
+  .blue-section {
+    .blue-section();
+
+    h1 {
+      color: @white;
+      font-size: @font-size-h1-big / 2;
+    }
+
+    h2 {
+      font-weight: bold;
+      letter-spacing: @medium-letter-spacing / 2;
+    }
+
+    @media screen and (max-width: @screen-md) {
+      .link {
+        margin: @vertical-small-spacing/4 0;
+      }
+
+      .header-spacing {
+        height: 0;
```

Comment:
Chiar nu înțeleg de ce ai scos height.

Deci am avut clasa de `header-spacing` cu margin-bottom la un moment dat. Dar mi-am dat seama ca e prost, asa ca am facut-o cu height, fara margin-bottom ca sa putem mai usor sa debugam in developer tools.